st2common/st2common/models/api/action.py

```diff
@@ -516,7 +516,8 @@ class ActionAliasAPI(BaseAPI, APIUIDMixin):
                 "type": "object",
                 "properties": {
                     "enabled": {"type": "boolean"},
-                    "format": {"type": "string"}
+                    "format": {"type": "string"},
+                    "append_url": {"type": "boolean"}
```

Comment:
Yes, same as enabled. We discussed it with @jfryman: URL is appended by default to everything (even a custom format string) unless explicitly set to "false".